app/helpers/controllers/saveOrderItem.js

```diff
@@ -0,0 +1,45 @@
+import { logger } from '../../logger';
+import { postOrderItem } from '../api/ordapi/postOrderItem';
+import { putOrderItem } from '../api/ordapi/putOrderItem';
+
+export const saveOrderItem = async ({
+  orderId,
+  orderItemId,
+  orderItemType,
+  accessToken,
+  serviceRecipientId,
+  serviceRecipientName,
+  itemId,
+  itemName,
+  selectedPrice,
+  formData,
+}) => {
+  try {
+    const response = orderItemId === 'neworderitem'
+      ? await postOrderItem({
+        accessToken,
+        orderId,
+        orderItemType,
+        serviceRecipientId,
+        serviceRecipientName,
+        itemId,
+        itemName,
+        selectedPrice,
+        formData,
+      })
+      : await putOrderItem({
+        accessToken,
+        orderId,
+        orderItemId,
+        formData,
+      });
+    return response;
+  } catch (err) {
+    if (err.response.status === 400 && err.response.data && err.response.data.errors) {
```

Comment:
I don't think you should be reshaping the response like this if there are errors. router.post('/:orderItemId', for additional services for example expect the errors to be in .errors

Reply:
I'm not sure what you mean here. But we are not reshaping the error response. It's just how the error response is returned to us from the api and axios. In this situation we want to handle when we get a 400 with errors back from the api. 

This is a pattern we've used for error handling the API validation errors through out the app so far. We can have a think about how we can do this better.

Reply:
ah, OK, I misread it, I thought it was returning just the array of errors, not { errors: [] }, which would completely mismatch.
You next layer up tests though are expecting { success: false, errors: [{}] }), which this function won't produce. I think you are getting away with it as the absence of success evaluates to false.

Reply:
Can you please change it to include success:false?